### DISCUSSION TITLE
Assets folder not included during `compile`

### DISCUSSION THREAD

**Discussion by codeit-ninja** [cawa-93/vite-electron-builder#847] [Q&A] [answered]
I compiled to app for local debugging and get errors in the console that all the images are missing because the assets folder doesn't exist.

![image](https://user-images.githubusercontent.com/22219122/194713322-1b85d727-6937-400a-836e-1af4fee97dde.png)

**Comment by codeit-ninja** [comment, marked as answer]
Ok, sorry for asking .. I had to look into the `vite` *config* and documentation.

You need to create an `public` folder in the root of the `renderer` package.
And then instead of importing it like 

```
<img src="./public/image.png" />
```

You have to do
```
<img src="./image.png" />
```

Vite puts everything from the public dir in the output root directory.

**Reply by cawa-93** [reply]
Best practices: https://vitejs.dev/guide/assets.html

**Reply by codeit-ninja** [reply]
Yes, but my problem originated from including images from css files.

**Reply by cawa-93** [reply]
I preffer explicit importing assets:
```vue
<script>
import AppBrandLogo from 'app-logo.png' // <--
</script>

<template>
<img :src="AppBrandLogo"/>
</template>
```
